api/receiver/api.go

```diff
@@ -78,6 +79,24 @@ func (a apiImpl) AddTestRun(testRun *shared.TestRun) (shared.Key, error) {
 	return key, nil
 }
 
+func (a apiImpl) IsAdmin(r *http.Request) bool {
+	logger := shared.GetLogger(a.Context())
+	acl, err := shared.NewGitHubAccessControlFromRequest(a, a.store, r)
+	if err != nil {
+		logger.Errorf("Error creating GitHubAccessControl: %s", err.Error())
+		return false
+	}
+	if acl == nil {
+		return false
```

Comment:
I am vaguely concerned about this being a silent failure mode (in terms of what happens one day when someone hits it), but maybe not having a cookie (which is currently the only way to hit this afaik) is a normal enough occurrence that logging it would be spammy.

Reply:
Yeah, this is the default (and the vast majority) case. `NewGitHubAccessControlFromRequest` has a comment documenting the `(nil, nil)` case which hopefully will give future people (or me) enough hints not to break the contract.